app.js

```diff
@@ -0,0 +1,48 @@
+const tool = require('./src/tool');
+
+let formbuilder = [
```

Comment:
Variable naming isn't following the proper convention, capitalize the "B", and you should be good.